pdf/pdfcanvas.cpp

```diff
@@ -187,6 +211,19 @@ int PDFCanvas::currentPage() const
     return d->currentPage;
 }
 
+bool PDFCanvas::rememberPagePosition() const
+{
+    return d->rememberPagePosition;
+}
+
+void PDFCanvas::setRememberPagePosition(bool status)
+{
+    if (d->rememberPagePosition != status) {
+        d->rememberPagePosition = status;
```

Comment:
I agree, but since the rememberPagePosition property is set when the PDFCanvas is built from QML, this worst case is currently not likely in the current PDFDocumentPage QML implementation. Indeed, when looking at pdfdocument.cpp source, one can see that loading is deferred until the PDF.Document QML object is completed. So it seems to me that when the document is loaded, the PDFCanvas object will already be completed also and its rememberPagePosition property already set.

I may be wrong though, what do you think ?